images/openstack-resource-controller.yml

```diff
@@ -0,0 +1,26 @@
+content:
+  source:
+    dockerfile: Dockerfile.rhel
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/openstack-resource-controller.git
+      web: https://github.com/openshift/openstack-resource-controller
+    ci_alignment:
+      streams_prs:
+        ci_build_root:
+          stream: rhel-9-golang-ci-build-root
+distgit:
+  branch: rhaos-{MAJOR}.{MINOR}-rhel-9
+  component: openstack-resource-controller-container
+enabled_repos:
+- rhel-9-appstream-rpms
+- rhel-9-baseos-rpms
+for_payload: false
+from:
+  builder:
+  - stream: rhel-9-golang
+  member: openshift-enterprise-base-rhel9
+name: openshift/openstack-resource-controller-rhel9
```

Comment:
shouldn't we add `ose-` prefix?

Reply:
Names can be double checked based on what we did here: https://gitlab.cee.redhat.com/hybrid-platforms/org/-/merge_requests/45/diffs#64cb79d1624d0229ed452d2b553b872c8f5777d0_0_7

Reply:
comparing to other images it is used here 
name: openshift/ose-$(image name)

whereas the  file name doesn't have to have the `ose` - in its name.